app/views/admin/users/new.html.haml

```diff
@@ -0,0 +1,13 @@
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    %h1.govuk-heading-xl
+      Add a new user
+    .ccs-user-form-group
+      = form_for [:admin, @user] do |form|
+        - @user.errors.full_messages.each do |message|
+          = message
+        %label.govuk-label{for: 'name'} Name
```

Comment:
For the fields to be given the correct names for Rails to understand it, you need to use the Rails form helpers, which you can add classes to as so:

```
        = form.label :name, class: 'govuk-label'
        = form.text_field :name, class: 'govuk-input'
        = form.label :email, 'Email address', class: 'govuk-label'
        = form.text_field :email, class: 'govuk-input'
        = form.submit 'Add new user', class: 'govuk-button'
```

Reply:
Thankyou, ill update that now